PerformanceNavigationTiming for User Agent Launch/explainer.md

```diff
@@ -71,76 +56,155 @@ unloadEventStart: 0
+This solution could also be easily extended to other performance structures if so desired. It also allows the user agent to decide what high vs low systemEntropy might mean, and potentially extend the enum later providing more granularity for consumers of the API (e.g., ‘medium’ or 'veryHigh’).  
 
-| HTML History Handling Type | Navigation Type |
-| -------------------------- | --------------- |
-| default                    | navigation      |
-| reload                     | reload          |
-| replace                    | back_forward    |
-| preload                    | prerender       |
+## Alternative Considered Solutions 
 
-In the text of the specification, we propose adding a new `historyHandling` type called **`launch`** that can be used in the _optional_ `historyHandling` parameter instead of the value `default` by default in the case where the browser is navigating from a cold-start scenario. We would propose ammending Step #6 to read:
-> If historyHandling is "default" **_or "launch"_**, and any of the following are true:
+### Add new NavigationType value for PerformanceNavigationTiming 
 
-Because the history handling property is marked as optional, this should make the new type interoperable with the default value of `default`. This would allow us to map the `navigate_during_user_agent_launch` type to the `launch` type specified in HTML.
+One considered proposal was to create a new value for the NavigationType enum to differentiate navigations during launch from others in measurements of web application performance. Launch was specifically selected as that is the most common use case. The new enum definition will appear as follows: 
+
+```enum NavigationType { 
+    "navigate", 
+    "reload", 
+    "back_forward", 
+    "prerender", 
+    "navigate_during_user_agent_launch", <== New 
+}; 
+```
+
+The payload of a performance.getEntriesByType("navigation") call would look like: 
+
+```connectEnd: 126.19999998807907 
+connectStart: 126.19999998807907 
+<snip> 
+transferSize: 300 
+type: "navigate_during_user_agent_launch" 
+unloadEventEnd: 0 
+unloadEventStart: 0 
+workerStart: 125.90000003576279 
+```
+
+An example of how a web application might use this: 
+
+```const navigationEntries = window.performance.getEntriesByType('navigation'); 
+let navigationType; 
+if (navigationEntries.length > 0) { 
+    const navigationEntry = navigationEntries[0]; 
+    navigationType = navigationEntry.type || 'UndefinedType'; 
+} else { 
+    navigationType = 'none'; 
+} 
+```
+
+However, these other navigation types can also occur, albeit less frequently, during a non-optimal performance state. It seems impractical to introduce a new enum for each existing navigation type, especially as it complicates the mapping to the historyHandling type. Also consider that as other cases become more common, we would be expanding the potential test matrix. 
+    
+### Add new Type value for performance.getEntriesByType 
```

Comment:
Is there an option to use 'launch' as a type and provide characteristics of the launch, that when combined with 'navigation' could be used to compute "true page load time"? Once concern with entropy is that we're discarding metrics and only keeping the ones where it's in optimal state, which could be rare. By contrast, providing the launch timing details would allow subtracting from total tile to find "true page time". Not sure if this is feasible engineering-wise, however.